tests/cortex/memory.x

```diff
@@ -6,6 +6,8 @@ MEMORY
 
 SECTIONS {
   linkme_SHENANIGANS : { *(linkme_SHENANIGANS) } > FLASH
+  linkm2_SHENANIGANS : { *(linkm2_SHENANIGANS) } > FLASH
```

Comment:
Good catch, these symbols got added in #53 which was after the cortex test got disabled.